driver/pace/driver/gridconfig.py

```diff
@@ -1,23 +1,23 @@
 import dataclasses
 from typing import Optional
 
-import numpy as np
-
 
 @dataclasses.dataclass
 class GridConfig:
```

Comment:
You should be able to delete this file, its functionality was moved to `grid.py`.